prow/scripts/provision-vm-and-start-busola-k3s.sh

```diff
@@ -27,6 +27,9 @@ cleanup() {
     
     # do not fail the job regardless of the vm deletion result
     set +e
+    
+    utils::receive_from_vm "${ZONE}" "busola-ui-test-${RANDOM_ID}" "$HOME/busola-tests/cypress/screenshots" "${ARTIFACTS}"
```

Comment:
Can we also add /cypress/videos?

Reply:
Yes but this would take much more space and probably a bit more time. @adamwalach what's your opinion on this?

Reply:
it depends, how much space this video consumes and is it present only in case of error?

Reply:
Yes, it's present only in case of error and should take 2-10MB depending on how many tests suites fail.

Reply:
Video is created for every spec file, now we have around 6 spec files and each movie has 5-10 MB. But videos are much better for debugging tests.

Reply:
https://github.com/kyma-project/busola/pull/170

Reply:
I added support for both screenshots and videos. Please have a look again @adamwalach @Sawthis

Reply:
looks good for me